test/0140/boolean-field-naming.test.js

```diff
@@ -0,0 +1,86 @@
+const { linterForAepRule } = require('../utils');
+require('../matchers');
+
+let linter;
+
+beforeAll(async () => {
+  linter = await linterForAepRule('0140', 'aep-140-boolean-field-naming');
+  return linter;
+});
+
+test('aep-140-boolean-field-naming should find warning', () => {
+  const oasDoc = {
+    openapi: '3.0.3',
```

Comment:
FYI we are moving to OAS 3.1.x: https://github.com/aep-dev/aeps/pull/287.

Reply:
Yes but we can do that in a separate PR. We need to fix all the existing rules too and may as well do them all together.